Tests/SourceKitLSPTests/ClosureCompletionFormatTests.swift

```diff
@@ -0,0 +1,129 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftBasicFormat
+import SwiftParser
+import SwiftSyntax
+import Swift
+@testable import SourceKitLSP
+import XCTest
+
+fileprivate func assertFormatted<T: SyntaxProtocol>(
+  tree: T,
+  expected: String,
+  using format: ClosureCompletionFormat = ClosureCompletionFormat(indentationWidth: .spaces(4)),
+  file: StaticString = #filePath,
+  line: UInt = #line
+) {
+  XCTAssertEqual(tree.formatted(using: format).description, expected, file: file, line: line)
```

Comment:
ℹ️ Unfortunately it doesn't seem like _SwiftSyntaxTestSupport can be added to the dependencies, so I had to use plain `XCTAssertEqual` instead of the nice `assertStringsEqualWithDiff`.